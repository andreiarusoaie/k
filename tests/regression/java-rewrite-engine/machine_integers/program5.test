 MyListItem(miMInt(uaddMInt(mi(32, -1), mi(32, -1)))),
 MyListItem(overflowMInt(uaddMInt(mi(32, -1), mi(32, -1))))

