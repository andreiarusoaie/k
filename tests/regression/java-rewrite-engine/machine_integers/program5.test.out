<k>
    MyListItem ( -2 ) , MyListItem ( true )
</k>
